test/Sentry.Tests/Protocol/TransactionTests.cs

```diff
@@ -388,6 +388,42 @@ public void Finish_SentryRequestSpansGetIgnored()
         transaction.Spans.Should().NotContain(s => s.Operation == "sentryRequest");
     }
 
+    [Fact]
+    public async Task Finish_SentryRequestTransactionGetsIgnored()
+    {
+        // Arrange
+        var client = Substitute.For<ISentryClient>();
+        var options = new SentryOptions
+        {
+            Dsn = ValidDsn,
+            IdleTimeout = TimeSpan.FromMilliseconds(2)
+        };
+        var hub = new Hub(options, client);
+        var context = new TransactionContext(
+            SpanId.Create(),
+            SpanId.Create(),
+            SentryId.Create(),
+            "my name",
+            "my operation",
+            "description",
+            SpanStatus.Ok,
+            null,
+            true,
+            TransactionNameSource.Component
+        );
+
+        var transaction = new TransactionTracer(hub, context)
+        {
+            IsSentryRequest = true
+        };
+
+        // Act
+        await Task.Delay(5);
```

Comment:
```suggestion
        await Task.Delay(TimeSpan.FromMilliseconds(5));
```
Nitpick just for consistency.